### ISSUE TITLE
Default self-signed certificate is generated, but expires immediately

### ISSUE THREAD

**Issue by mialam24** [containous/traefik#3168]
### Do you want to request a *feature* or report a *bug*?
bug

### What did you do? 
I brought up a Docker stack, including Traefik and configured the entrypoint as follows:
```toml
[entryPoints]
  [entryPoints.http]
  address = ":80"
    [entryPoints.http.redirect]
    entryPoint = "https"
  [entryPoints.https]
  address = ":443"
    [entryPoints.https.tls]
```
**Note** that the TLS configuration is left empty and Let's Encrypt is not used

### What did you expect to see?
I expected to browse to one of the https frontends and see a valid self-signed certificate

### What did you see instead?
I saw a certificate named TRAEFIK DEFAULT CERT, but was expired. Using a Chrome browser, the 'Not Valid Before' and 'Not Valid After' times were identical

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
$ docker run traefik:1.6 version
Version:      v1.6.0-rc4
Codename:     tetedemoine
Go version:   go1.10.1
Built:        2018-04-04_01:40:48PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
[docker]
domain = "docker.local"
swarmmode = true

[api]
debug = true

[entryPoints]
  [entryPoints.http]
  address = ":80"
    [entryPoints.http.redirect]
    entryPoint = "https"
  [entryPoints.https]
  address = ":443"
    [entryPoints.https.tls]
```

### If applicable, please paste the log output at DEBUG level (`--logLevel=DEBUG` switch)

```
(paste your output here)
```

**Comment by SantoDE** [comment]
Hey @mialam24 ,

thanks for your report. I've raised a PR #3169 to fix that issue :)

**Comment by traefiker** [comment]
Closed by #3169.

**Comment by ldez** [comment]
Fixed in 1.6.0-rc5
